### PR TITLE
tpm_verify_device: Update some TPM parameters

### PR DESCRIPTION
**Pull request by PaulYuuu**
Deps on avocado-framework/avocado-vt/pull/3364, Update
global TPM parameters and set different TPM names in
different VMs.

ID: 2072812
Signed-off-by: Yihuang Yu <yihyu@redhat.com>

### CODE REVIEW

**Comment by PaulYuuu** [comment]
Hello @qcheng-redhat, could you help to review this patch? Thanks in advance.

File: qemu/tests/cfg/tpm_verify_device.cfg [hunk 1]
```diff
@@ -31,13 +31,13 @@
...
+            tpm_type = emulator
+            tpm_model = tpm-crb
             ppc64le, ppc64:
-                tpm_model_tpm0 = tpm-spapr
+                tpm_model = tpm-spapr
             aarch64:
-                tpm_model_tpm0 = tpm-tis-device
-            tpm_version_tpm0 = 2.0
+                tpm_model = tpm-tis-device
+            tpm_version = 2.0
```
Comment (qcheng-redhat) [inline qemu/tests/cfg/tpm_verify_device.cfg:40]: Seems we have a dup tpm_version here ?

Comment (PaulYuuu) [reply qemu/tests/cfg/tpm_verify_device.cfg:40]: Yes, it's a history issue, I will delete it.

Comment (qcheng-redhat) [reply qemu/tests/cfg/tpm_verify_device.cfg]: Thanks for fixing the history issue.

Others LGTM.

 (2/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.q35: STARTED
 (2/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.q35: PASS (95.04 s)
 (3/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_check_ovmf.q35: STARTED
 (3/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_check_ovmf.q35: PASS (94.87 s)
 (4/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_reboot.q35: STARTED
 (4/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_reboot.q35: PASS (903.66 s)
 (5/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_multi_vms.q35: STARTED
 (5/5) Host_RHEL.m9.u0.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.Win11.x86_64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_multi_vms.q35: PASS (222.42 s)

**Comment by zhenyzha** [comment]
```
 (1/4) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.tpm_verify_device.with_emulator.arm64-pci: STARTED
| (1/4) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.tpm_verify_device.with_emulator.arm64-pci: PASS (41.45 s)
 (2/4) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_check_aavmf.arm64-pci: STARTED
\ (2/4) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_check_aavmf.arm64-pci: PASS (38.33 s)
 (3/4) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_reboot.arm64-pci: STARTED
| (3/4) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_reboot.arm64-pci: PASS (615.79 s)
 (4/4) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_multi_vms.arm64-pci: STARTED
| (4/4) Host_RHEL.m9.u0.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.tpm_verify_device.with_emulator.with_multi_vms.arm64-pci: PASS (139.41 s)
```

With https://github.com/avocado-framework/avocado-vt/pull/3364 , all pass on arm.

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Comment by PaulYuuu** [comment]
Merge this one since the dependency patch is merged.

**Pull request merged**